sdk/samples/iot/hub/src/paho_iot_hub_pnp_example.c

```diff
@@ -0,0 +1,752 @@
+
+  {
+    printf("Failed to get MQTT clientId, return code %d\n", rc);
+    return rc;
+  }
+
+  printf("%s\r\n", mqtt_username);
+
+  // This sample uses X509 authentication so the password field is set to NULL
+  mqtt_connect_options.username = mqtt_username;
+  mqtt_connect_options.password = NULL;
+
+  // Set the device cert for tls mutual authentication
+  mqtt_ssl_options.keyStore = (char*)x509_cert_pem_file;
+  if (*x509_trust_pem_file != '\0')
+  {
+    mqtt_ssl_options.trustStore = (char*)x509_trust_pem_file;
+  }
+
+  mqtt_connect_options.ssl = &mqtt_ssl_options;
+
+  // Connect to IoT Hub
+  if ((rc = MQTTClient_connect(mqtt_client, &mqtt_connect_options)) != MQTTCLIENT_SUCCESS)
+  {
+    printf("Failed to connect, return code %d\n", rc);
+    return rc;
+  }
+
+  return 0;
+}
+
+static int subscribe()
+{
+  int rc;
+
+  // Subscribe to the methods topic. Messages received on this topic are methods to be invoked
+  // on the device.
+  if ((rc = MQTTClient_subscribe(mqtt_client, AZ_IOT_HUB_CLIENT_METHODS_SUBSCRIBE_TOPIC, 1))
+      != MQTTCLIENT_SUCCESS)
+  {
+    printf("Failed to subscribe to twin patch topic filter, return code %d\n", rc);
+    return rc;
+  }
+
+  // Subscribe to the desired properties PATCH topic. Messages received on this topic will be
+  // updates to the desired properties.
+  if ((rc = MQTTClient_subscribe(mqtt_client, AZ_IOT_HUB_CLIENT_TWIN_PATCH_SUBSCRIBE_TOPIC, 1))
+      != MQTTCLIENT_SUCCESS)
+  {
+    printf("Failed to subscribe to twin patch topic filter, return code %d\n", rc);
+    return rc;
+  }
+
+  // Subscribe to the twin response topic. Messages received on this topic will be response statuses
+  // from published reported properties or the requested twin document from twin GET publish
+  // messages
+  if ((rc = MQTTClient_subscribe(mqtt_client, AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_SUBSCRIBE_TOPIC, 1))
+      != MQTTCLIENT_SUCCESS)
+  {
+    printf("Failed to subscribe to twin response topic filter, return code %d\n", rc);
+    return rc;
+  }
+
+  return 0;
+}
+
+// Send JSON formatted telemetry messages
+static int send_telemetry_messages()
+{
+  int rc;
+
+  if (az_failed(
+          rc = az_iot_hub_client_telemetry_get_publish_topic(
+              &client, NULL, telemetry_topic, sizeof(telemetry_topic), NULL)))
+  {
+    return rc;
+  }
+
+  az_json_builder json_builder;
+  az_result result;
+  result = az_json_builder_init(&json_builder, AZ_SPAN_FROM_BUFFER(telemetry_payload), NULL);
+  result = az_json_builder_append_begin_object(&json_builder);
+  result = az_json_builder_append_property_name(&json_builder, telemetry_name);
+  result = az_json_builder_append_int32_number(&json_builder, (int32_t)current_device_temp);
+  result = az_json_builder_append_end_object(&json_builder);
+  az_span json_payload = az_json_builder_get_json(&json_builder);
+  (void)result;
+
+  // New line to separate messages on the console
+  printf("\n");
```

Comment:
>printf [](start = 2, length = 6)

Lets use the same pattern everywhere, either use putch or printf to put \n character 

line 621